File: web/src/model/victron_controller/dashboard/from_0_2_0_wsclientmessage_send-command.ts
// @ts-nocheck
import {SendCommand as v0_2_0_SendCommand} from './v0_2_0/WsClientMessage'
import {SendCommand as dashboard_SendCommand} from './WsClientMessage'

export function convert__ws_client_message__send_command__from__0_2_0(from: v0_2_0_SendCommand): dashboard_SendCommand {
    return new dashboard_SendCommand (
        JSON.parse(JSON.stringify(from.body))
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_wsclientmessage_send_command.rs


pub fn convert__wsclientmessage__send_command__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::ws_client_message::SendCommand) -> crate::victron_controller::dashboard::ws_client_message::SendCommand {
    crate::victron_controller::dashboard::ws_client_message::SendCommand {
        body: serde_json::from_value(serde_json::to_value(&from.body).unwrap()).unwrap(),
    }
}